test/linalg/diagonal.jl

```diff
@@ -91,6 +91,8 @@ srand(1)
                     Uc = ctranspose(U)
                     target = scale!(Uc,inv.(D.diag))
                     @test A_rdiv_B!(Uc,D) ≈ target atol=2n^3*eps(relty)*(1+(elty<:Complex))
+                    @test_throws DimensionMismatch A_rdiv_B!(eye(elty, n-1),D)
```

Comment:
I spaced out the commas all around this and set the tolerances as their own variables so we can change them all together (and the wrapping is better)...